Makefile

```diff
@@ -2,14 +2,14 @@
 # Copyright (c) The mldsa-native project authors
 # SPDX-License-Identifier: Apache-2.0 OR ISC OR MIT
 
-.PHONY: func kat acvp stack \
+.PHONY: func kat acvp stack keccak \
```

Comment:
Can we use the same naming as in mlkem-native and call this unit: https://github.com/pq-code-package/mlkem-native/blob/9ae2223e835c1abcc8d6857dd3b7f8ce00a05216/Makefile#L84? 

If we can keep the diff to mlkem-native minimal that would be appreciated.